src/apps/backend/modules/share-task-request/rest-api/share-task-request-server.ts

```diff
@@ -0,0 +1,12 @@
+import { ApplicationServer } from '../../application';
+
+import ShareTaskRequestRouter from './share-task-request-router';
+
+export default class ShareTaskRequestServer extends ApplicationServer {
+  configure(): void {
+    const { server } = this;
+    const router = new ShareTaskRequestRouter();
+
+    server.use('/share-task-request', router.router);
```

Comment:
`/tasks/:taskId/share-task-requests`

share-task-requests is not an entity in the system.
The entity continue to be task.